src/tests/tic-tac-toe.test.ts

```diff
@@ -0,0 +1,21 @@
+import Game from "../tic-tac-toe";
+
+describe("Tic-tac-toe", () => {
+  let game: Game;
+  beforeEach(() => {
```

Comment:
I would typically separate `beforeEach`, `it`, etc. with newlines to make tests a little more readable:

```js
describe(() => {
  beforeEach()
  
  it()

  it()
})
```

Reply:
Agreed! Ended up doing this after the PR.